ASM/c/kaleido_scope.h

```diff
@@ -0,0 +1,365 @@
+#pragma once
```

Comment:
There is stuff here that should definitely be in other files or already is (https://github.com/OoTRandomizer/OoT-Randomizer/blob/Dev/ASM/c/audio.h). Please do not overload .h files. We do not need 50 definitions of the same thing in 50 different files. Try to put them in other header files that make sense.